starter-code/index.html

```diff
@@ -44,8 +44,8 @@
         <li>$1 pepperonni</li>
         <li>$1 mushrooms</li>
         <li>$1 green peppers</li>
-        <li>$3 white sauce</li>
-        <li>$5 gluten-free crust</li>
+        <li class="li-white-sauce">$<spans>3</span> white sauce</li>
+        <li class="li-gluten-free-crust">$5 gluten-free crust</li>
```

Comment:
This a valid way to select specific li's from a list but we have to think about the extreme cases where there may be 10s, if not 100's of list items in a list; in which case adding an ID or unique class is not practical. A better tool to use would be the advanced css selector, nth-child.

Ex: $('.price > ul > li:nth-child(1)').toggle();